build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -101,7 +101,7 @@ steps:
 
   - script: |
       set -e
-      ./scripts/test.sh --build --coverage --reporter mocha-junit-reporter
+      ./scripts/test.sh --build --tfs "Unit Tests"
```

Comment:
Why remove coverage here? That's still valuable to see

Reply:
Just testing, I'll just close this PR till im done testing.